src/Api/Utilities/ApiHelpers.cs

```diff
@@ -29,5 +34,47 @@ public async static Task<T> ReadJsonFileFromBody<T>(HttpContext httpContext, IFo
 
             return obj;
         }
+
+        /// <summary>
+        /// Validates Azure event subscription and calls the appropriate event handler. Responds HttpOk.
+        /// </summary>
+        /// <param name="request">HttpRequest received from Azure</param>
+        /// <param name="eventTypeHandlers">Dictionary of eventType strings and their associated handlers.</param>
+        /// <returns>OkObjectResult</returns>
+        // Reference https://docs.microsoft.com/en-us/azure/event-grid/receive-events
```

Comment:
```suggestion
        /// <remarks>Reference https://docs.microsoft.com/en-us/azure/event-grid/receive-events</remarks>
```
Add this comment to a `/// <remarks></remarks>` tag. Keeps the XML documentation comments bumped against the method definition.